pkg/cri/server/helpers.go

```diff
@@ -108,11 +108,11 @@ func makeSandboxName(s *runtime.PodSandboxMetadata) string {
 // unique.
 func makeContainerName(c *runtime.ContainerMetadata, s *runtime.PodSandboxMetadata) string {
 	return strings.Join([]string{
-		c.Name,                       // 0: container name
-		s.Name,                       // 1: pod name
-		s.Namespace,                  // 2: pod namespace
-		s.Uid,                        // 3: pod uid
-		fmt.Sprintf("%d", c.Attempt), // 4: attempt number of creating the container
+		c.Name,      // 0: container name
+		s.Name,      // 1: pod name
+		s.Namespace, // 2: pod namespace
+		s.Uid,       // 3: pod uid
+		strconv.FormatUint(uint64(s.Attempt), 10), // 4: attempt number of creating the container
```

Comment:
Oh!!! I think I see it 🤦 🙈 🙈 🙈 

```suggestion
		strconv.FormatUint(uint64(c.Attempt), 10), // 4: attempt number of creating the container
```

Reply:
FOR REALZ! So many reviewers (including "myself") and very much missing this!

<img width="80" alt="Screenshot 2023-08-23 at 17 20 54" src="https://github.com/containerd/containerd/assets/1804568/134e30eb-cf9c-469e-82c3-0eb301f52324">